### ISSUE TITLE
TAF and other aspects that should be mentioned in the adjoint tutorials

### ISSUE THREAD

**Issue by gaelforget** [ECCO-Hackweek/ecco-2024#135]
In https://github.com/ECCO-Hackweek/ecco-2024/blob/main/book/tutorials/AD_tuts_index.md it would be useful to add, @Shreyas911 

- a presentation of TAF -- the tool generally used today for getting the MITgcm adjoint in ECCO and beyond. 
- pointers to the relevant MITgcm docs
- links to the EMU part that is based on TAF

During that session, maybe @owang01 could demo TAF as an explanation for how EMU is built?

**Comment by Shreyas911** [comment]
I agree @gaelforget, it's just that I have never used TAF, so I don't know much about it. Someone more experienced with TAF could take this on if possible.
